BookingProject/BookingWebAPI/Migrator/Program.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Migrator.Interfaces;
+using Migrator.Services;
+using BookingWebApi.Infrastructure.Data;
+using BookingWebApi.Domain.Entities;
+using BookingWebApi.Application.Decorators;
+using AutoMapper;
+using BookingWebApi.Application.Interfaces;
+
+namespace Migrator
+{
+    class Program
+    {
+        static async Task Main(string[] args)
+        {
+            var host = CreateHostBuilder(args).Build();
+
+            var migrationService = host.Services.GetRequiredService<IDataMigrationService>();
+
+            var result = await migrationService.MigrateData("Company.json");
+
+            if (result.IsSuccess)
+            {
+                Console.WriteLine("Data migration completed successfully.");
+            }
+            else
+            {
+                Console.WriteLine($"Data migration failed: {result.ErrorMessage}");
+            }
```

Comment:
Use `ILogger<T>` for logging. You might want to publish migration logs to Logstash, Coralogix, store them to file, etc, but doing your way we cannot do it

Reply:
Fixed